action.yml

```diff
@@ -199,6 +199,26 @@ runs:
           cd /tmp/skaffold
           skaffold config set collect-metrics false
           skaffold deploy
+
+          if [ $? -eq 0 ]; then
+            # Publish Artifact Info on success
+            response=$(curl -i -X 'POST' "$CLOUDBEES_API_URL/v2/workflows/runs/artifactinfos" \
+              -d "{
+                \"runId\": \"$CLOUDBEES_RUN_ID\",
+                \"runAttempt\": \"$CLOUDBEES_RUN_ATTEMPT\",
+                \"name\": \"$RELEASE_NAME\",
+                \"url\": \"$CHART_LOCATION\",
+                \"version\": \"$CHART_VERSION\",
+                \"type\": \"deploy\"
+              }" \
+              -H "Authorization: Bearer $CLOUDBEES_API_TOKEN" \
+              -H 'Content-Type: application/json') || command_failed=1
+
+            # Check if the command failed
+            if [ ${command_failed:-0} -eq 1 ]; then
+              echo "WARNING: Publish Artifact Info failed with response '$response'"
```

Comment:
Minor: we can use similar wordings as `kaniko` action - https://github.com/cloudbees-io/kaniko/pull/42/files#diff-fc3fda5e510f476957b63b19fef2fc05271e4d179a1ad11b10bb88ee8aacd7feR73 for consistency.

`WARN: failed to save artifact information: '$response'`